imap.go

```diff
@@ -60,13 +60,14 @@ const (
 	MailboxAttrRemote        MailboxAttr = "\\Remote"
 
 	// Role (aka. "special-use") attributes
-	MailboxAttrAll     MailboxAttr = "\\All"
-	MailboxAttrArchive MailboxAttr = "\\Archive"
-	MailboxAttrDrafts  MailboxAttr = "\\Drafts"
-	MailboxAttrFlagged MailboxAttr = "\\Flagged"
-	MailboxAttrJunk    MailboxAttr = "\\Junk"
-	MailboxAttrSent    MailboxAttr = "\\Sent"
-	MailboxAttrTrash   MailboxAttr = "\\Trash"
+	MailboxAttrAll       MailboxAttr = "\\All"
+	MailboxAttrArchive   MailboxAttr = "\\Archive"
+	MailboxAttrDrafts    MailboxAttr = "\\Drafts"
+	MailboxAttrFlagged   MailboxAttr = "\\Flagged"
+	MailboxAttrJunk      MailboxAttr = "\\Junk"
+	MailboxAttrSent      MailboxAttr = "\\Sent"
+	MailboxAttrTrash     MailboxAttr = "\\Trash"
+	MailboxAttrImportant MailboxAttr = "\\Important" // RFC 8457
```

Comment:
`\Important` is not a role ("special-use") attribute, so can we add a blank line in-between `MailboxAttrTrash` and `MailboxAttrImportant` to make this clear?

Reply:
Hm, actually, it is… Nevrmind!